sqlglot/schema.py

```diff
@@ -71,7 +71,7 @@ def column_names(
     def get_column_type(
         self,
         table: exp.Table | str,
-        column: exp.Column,
+        column: exp.Column | str,
```

Comment:
The previous type hint was more restrictive than needed, we already supported strings in the implementation, e.g. as shown in this test: https://github.com/tobymao/sqlglot/blob/main/tests/test_schema.py#L167